docs/Downloading-Controlled-Access-Data.md

```diff
@@ -0,0 +1,48 @@
+# Downloading Controlled-Access Data
+
+## [Introduction](introduction)
+A user authentication token (data access token) is required for downloading controlled-access data programmatically from the cBioPortal. See [Obtaining an Authentication Token](#obtaining-an-authentication-token) for instructions on how to download a token and [Using Authentication Tokens](#using-authentication-tokens) for how to use it.
+
```

Comment:
Other pages in the documentation are in the style of a guild/tutorial to the cBioPortal deployer, rather than the user. They step the reader through the steps of how to configure and deploy the portal, so I think this page should try to take the same approach. I think the first section would guide the user on the choices available in portal.properties and give advice on how to select a token format, and then tell them what other settings are available. This could be organized property by property, or having a subsection for UUID configuration, a subsection for JWT configuration, and a subsection for the "none" mode - disallowing DAT in this portal instance.